js/templates/modals/settings/moderation.html

```diff
@@ -87,6 +77,16 @@ <h2 class="h3 clrT">Moderation</h2>
                   <% }); %>
                 </select>
               </div>
+              <div class="col2 js-feePercentageInput <% if (ob.fee && ob.fee.feeType === 'FIXED') { print('visuallyHidden') } %>">
+                <div class="inputPercentWrapper clrBr clrSh2">
+                  <% var perc = ob.fee.percentage || 0; %>
```

Comment:
Why is this necessary? Isn't this happening anyway with the default set to 0 in the model?